app/bundles/ReportBundle/Config/config.php

```diff
@@ -123,6 +123,10 @@
                 'alias'     => 'aggregator',
                 'arguments' => 'translator',
             ],
+            'mautic.form.type.report.settings' => [
+                'class' => 'Mautic\ReportBundle\Form\Type\ReportSettingsType',
```

Comment:
Lets use `\Mautic\ReportBundle\Form\Type\ReportSettingsType::type` instead of strings for the new code. It helps navigating in editors and it will throw more descriptive errors if there will be a typo in it.